mne/data/html_templates.py

```diff
@@ -100,3 +100,38 @@
     </tr>
 </table>
 """)
+
+evoked_template = Template("""
+<table class="table table-hover">
+    <tr>
+        <th>Content</th>
```

Comment:
I'm starting to think that maybe we should just call this `Conditions`? WDYT?

Reply:
I think the original `Comment` is also an option :)